packages/bits/spec/virtual-camera/types.ts

```diff
@@ -13,6 +13,7 @@ export const CAMERA_DEFAULT_SETTINGS: ICameraSettings = {
     currentSuiteName: "",
     currentTestName: "",
     globalLens: "percy",
+    responsiveCallback: undefined,
 }
 
 export const PERCY_DEFAULT_CONFIG: SnapshotOptions = {
```

Comment:
It might be good to separate out the constants into a separate constants.ts file.